docs/dev/dev_environment.md

```diff
@@ -316,7 +316,7 @@ When trying to debug an issue, one helpful thing you can look at are the logs wi
 !!! info
     Want to limit the log output even further? Use the `--tail <#>` command line argument in conjunction with `-f`.
 
-So for example, our plugin is named `nautobot-floor-plan`, the command would most likely be `docker logs nautobot_floor_plan_nautobot_1 -f`. You can find the name of all running containers via `docker ps`.
+So for example, our app is named `nautobot-floor-plan`, the command would most likely be `docker logs nautobot_floor_plan_nautobot_1 -f`. You can find the name of all running containers via `docker ps`.
```

Comment:
Would be a nice enhancement to the cookiecutter (if not already implemented) to add an `invoke logs` command as a wrapper around this.

Reply:
It's part of the cookie now ;-)